pkg/controllers/csi/gc/config.go

```diff
@@ -9,21 +9,21 @@ import (
 var (
 	log = logger.Get().WithName("csi-gc")
 
-	reclaimedMemoryMetric = prometheus.NewCounter(prometheus.CounterOpts{
+	reclaimedMemoryMetric = prometheus.NewCounter(prometheus.CounterOpts{ //nolint:promlinter
```

Comment:
I'm thinking of not enabling this linter at all. Because sounds like we enable it and only code we use prom related code we ignore it :)

Reply:
Yes i thought so as well. We can enable it but in my opinion it is unnecessary... and it would make the code ugly IMO. what do you think?

Reply:
I removed promlinter because as you already mentioned it is currently not necessary
https://github.com/Dynatrace/dynatrace-operator/pull/2710/commits/e48ccddd39950f46fa5cdcbb5a023832539d1f93